usecase/usecase-webapi-xwalk-tests/steps/SetCookie/step.js

```diff
@@ -0,0 +1,8 @@
+var step = '<font class="fontSize">'
+         + '<p>Test Purpose: </p>'
+         + '<p>Verifies HTTP response header "Set-Cookie".</p>'
+         + '<p>Test steps: </p>'
+         + '<p>Check the "Set-Cookie" value.</p>'
+         + '<p>Expected Result: </p>'
+         + '<p>Test passes if it should get the response header value expect "Set-Cookie", the value of "Set-Cookie" should be blank(null).</p>'
```

Comment:
Actually in short the expected result is "Set-Cookie response header value is blank(null)", right?

Tester may confuse here. Does the "blank(null)" means nothing or a string? Why the Set-Cookie value is so strange?

https://en.wikipedia.org/wiki/HTTP_cookie#Setting_a_cookie has an example like this

```
HTTP/1.0 200 OK
Content-type: text/html
Set-Cookie: theme=light
Set-Cookie: sessionToken=abc123; Expires=Wed, 09 Jun 2021 10:18:14 GMT
...
```

Can we make the value something human readable and meaningful?